Userland/Utilities/ping.cpp

```diff
@@ -161,14 +165,13 @@ int main(int argc, char** argv)
             total_pings++;
 
         for (;;) {
-            // FIXME: IPv4 headers are not actually fixed-size, handle other sizes.
-            auto pong_packet_result = ByteBuffer::create_uninitialized(sizeof(struct ip) + sizeof(struct icmphdr) + payload_size);
+            auto pong_packet_result = ByteBuffer::create_uninitialized(sizeof(struct ip) + max_optional_header_size_in_bytes
+                + sizeof(struct icmphdr) + payload_size);
```

Comment:
We usually don't split lines like this, how about:
```suggestion
            auto pong_packet_result = ByteBuffer::create_uninitialized(
                sizeof(struct ip) + max_optional_header_size_in_bytes + sizeof(struct icmphdr) + payload_size);
```